x509util/crlcheck/crlcheck.go

```diff
@@ -19,67 +19,203 @@ package main
 import (
 	"flag"
 	"fmt"
+	"io/ioutil"
+	"net/http"
 	"os"
+	"time"
 
+	"github.com/golang/glog"
 	"github.com/google/certificate-transparency-go/x509"
 	"github.com/google/certificate-transparency-go/x509util"
 )
 
-var caFile = flag.String("ca", "", "CA certificate file")
-var verbose = flag.Bool("verbose", false, "Verbose output")
+var (
+	caFile      = flag.String("ca", "", "CA certificate file")
+	strict      = flag.Bool("strict", false, "Strict validation of CRL contents")
+	expectCerts = flag.Bool("cert", false, "Input files are certificates not CRLs")
+)
 
 func main() {
 	flag.Parse()
+
+	// Build a list of possible CA certs from command line arguments.
 	var caCerts []*x509.Certificate
 	if *caFile != "" {
 		caDataList, err := x509util.ReadPossiblePEMFile(*caFile, "CERTIFICATE")
 		if err != nil {
-			fmt.Fprintf(os.Stderr, "%s: Failed to read CA cert data: %v\n", *caFile, err)
+			glog.Exitf("%s: Failed to read CA cert data: %v", *caFile, err)
```

Comment:
consistency nit: "%s: failed" to match Exitf text case below

Reply:
Done.